src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/avoidescapedunicodecharacters/Example3.java

```diff
@@ -0,0 +1,31 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="AvoidEscapedUnicodeCharacters">
+      <property name="allowByTailComment" value="true"/>
+    </module>
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.avoidescapedunicodecharacters;
+
+// xdoc section -- start
+public class Example3 {
+  // OK, a normal String below
+  String unitAbbrev = "μs";
+  // violation below
+  String unitAbbrev1 = "\u03bcs";
+  // violation below
+  String unitAbbrev2 = "\u03bc\u03bc\u03bc";
+  // violation below
+  String unitAbbrev3 = "\u03bcs";
+  String unitAbbrev4 = "\u03bc\u03bcs"; // it is  μs
```

Comment:
Please put comment above
`// ok, because there is trailing comment and allowByTailComment=true

Position on lines Should stay the same content of comments is changing only.